src/jstests/formio/components/textfield.spec.js

```diff
@@ -72,7 +72,7 @@ describe('TextField Component', () => {
     }).catch(done);
   });
 
-  test('TextField with address prefill refreshes street on invalid data', (done) => {
+  test('TextField with address prefill refreshes street if already filled', (done) => {
```

Comment:
* Please make it explicit that this only applies to editable textfields
* The behaviour is now the opposite of what the test name is - it should NOT refresh the street